api/proto/printable.proto

```diff
@@ -23,6 +23,9 @@ message PaymentRequest {
 
     /// Token id to transact in.
     uint64 token_id = 4;
+
+    /// Payment Id
+    uint64 payment_id = 5;
```

Comment:
:question: I'm guessing from the rest of this proto that using `uint64` over `fixed64` isn't a concern of leaking information via message size?

Reply:
whats the reason to use fixed64 ? Im not familiar with that type.

Reply:
fixed 64 will always be eight bytes in the proto message, while uint64 will use as few bytes as possible to represent the value.
My understanding is that the consistent(fixed) sizes can be used inside of encrypted messages to avoid leaking possible size ranges, if the information should be kept private.
Based on the other fields in the message using `fixed` here probably won't gain anything.

Reply:
ok, fixed sounds better, but we should change all of them, and then thats a higher impact change. Lets keep as UInt64.